.envTemplate

```diff
@@ -0,0 +1,18 @@
+// Copy this template to your own local machine, and name it, ".env.local" DO NOT ADD THIS FILE TO ".gitignore". The actual .env.local file should already be in ".gitignore" file already
+
+// Firebase needs these to initialize fire base app
+ REACT_APP_FIREBASE_API_KEY=""
```

Comment:
huh?

this exposes my firebase API info I didn't add this yet and the app seems to work just fine for rn after I pulled your code.

Reply:
I'm not getting what do you mean expose? That file(.envTemplate)is simply a template. When you create a copy of that template and name it ".env.local" into your local machine version of the code,you then add your actual values in that new copy. How would it get expose if the ".env.local" file is in .gitignore file already?

In the original sample of the code from the Firebase docs(e.g Firebase.js), although for demo purposes, it showed the cofigs settings, which included the fb api key, and other important key values in the file. If I were to commit that file, then it would be exposed.